unetsocket/c/samples/bbrecord.c

```diff
@@ -0,0 +1,80 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Script to record a signal.
+//
+// In terminal window (an example):
+//
+// $ make samples
+// $ ./bbrecord <ip_address> [port]
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "../unet.h"
+#include "../unet_ext.h"
+#include <math.h>
+
+#ifndef _WIN32
+#include <unistd.h>
+#include <netdb.h>
+#include <sys/time.h>
+#endif
+
+#define SIGLEN  48000*6 // change this as per the need
```

Comment:
General comment, if we are implementing calculations as macros, it is recommended to have brackets around them as preprocessors simply replace the values and can cause calculation errors.

Reply:
Fixed in fe57ea5fb25793077ec4931b35ed04822731fefc